### PR TITLE
Fixes SOL-1554 and SOL-1555

### PR DESCRIPTION
**Pull request by dsego**
Issues in adding a course ID (SOL-1554) 
Total Paid field is empty (SOL-1555)

Screen recording: https://vid.me/e/ga8T

### CODE REVIEW

**Comment by openedx-webhooks** [comment]
Thanks for the pull request, @dsego! It looks like you're a member of a company that does contract work for edX. If you're doing this work as part of a paid contract with edX, you should talk to edX about who will review this pull request. If this work is not part of a paid contract with edX, then you should ensure that there is an OSPR issue to track this work in [JIRA](https://openedx.atlassian.net), so that we don't lose track of your pull request.

To automatically create an OSPR issue for this pull request, just visit this link: https://openedx-webhooks.herokuapp.com/github/process_pr?repo=edx%2Fecommerce&number=483

**Comment by dsego** [comment]
@vkaracic @marjev Please take a look.

File: ecommerce/static/templates/coupon_form.html [hunk 1]
```diff
@@ -54,7 +54,7 @@
             <label for="benefit_value"><%= gettext('Discount Value') %></label>
             <div class="input-group">
                 <div class="benefit-addon input-group-addon"></div>
-                <input type="text" class="form-control" name="benefit_value">
+                <input type="number" step="any" class="form-control" name="benefit_value">
```
Comment (marjev) [inline ecommerce/static/templates/coupon_form.html]: why set step to `any`?

File: ecommerce/static/templates/coupon_form.html [hunk 2]
```diff
@@ -81,7 +81,7 @@
             <label for="price"><%= gettext('Total Paid') %></label>
             <div class="input-group">
                 <div class="input-group-addon">$</div>
-                <input type="text" class="form-control" name="price">
+                <input type="number" step="any" class="form-control" name="price">
```
Comment (marjev) [inline ecommerce/static/templates/coupon_form.html]: same question

**Comment by marjev** [comment]
other than two small questions, :+1:

File: ecommerce/static/js/models/coupon_model.js [hunk 3]
```diff
@@ -42,18 +43,12 @@ define([
                 title: { required: true },
                 client_username: { required: true },
                 // seat_type is for validation only, stock_record_ids holds the values
-                price: {
```
Comment (vkaracic) [inline ecommerce/static/js/models/coupon_model.js:45]: Why did you remove this validation?

Comment (dsego) [inline ecommerce/static/js/models/coupon_model.js:45]: Per sol-1555 the field can be left blank. Since you need the field to be non-empty for the backend, I will bring back the validation and add a default value.

**Comment by mjfrey** [comment]
Looks good.

:+1:  Rebase and squash
